R/LinReg.R

```diff
@@ -1,6 +1,6 @@
 LinReg<- function(model, particles=1000, plot=FALSE) {
 
-    data(radiata)
+    data(radiata, envir = environment())
```

Comment:
We can try. I sort-of rediscover / relearn this every time.  All I know right now is that I stick it into the `data/` directory and things but it has been a while since I added data to a package...  Just checked one of the recent ones and yes, I do use `LazyData: true` there and need no `environment()`.  Maybe that is just it.